dts/x86/intel/raptor_lake.dtsi

```diff
@@ -214,6 +214,11 @@
 			interrupts = <PCIE_IRQ_DETECT IRQ_TYPE_LOWEST_LEVEL_LOW 3>;
 			interrupt-parent = <&intc>;
 			current-speed = <115200>;
+			dma0: dma {
```

Comment:
Is the goal to use a dma with this uart or does this uart have what appears to be a dedicated DMA? this is a little bit different than whats typical.

Typically the node would specify the dma, channel(s), and maybe bus arbitration priority to use

e.g. 

```dts
uart0 { 
   dmas = <&dma0 3 0> <&dma0 4 0>;
   dma-names = "tx", "rx";
}
```

I'd like to understand a little better how this is going to be used by the uart driver in this instance.

Otherwise since this seems directly tied to the uart and likely the code changes needed for it aren't here, postpone the dtsi changes until the uart changes come in so the context is clearer.